src/app/main/main.component.js

```diff
@@ -1,6 +1,7 @@
 import 'babel/external-helpers';
 import angular from 'angular';
 import 'angular-ui-router';
+import 'angular-animate';
```

Comment:
@OzzieOrca , angular-animate is required by bootstrap ui to make smooth transitions. Otherwise they are choppy.